state/state_test.go

```diff
@@ -59,7 +59,7 @@ func TestMain(m *testing.M) {
 	}
 	defer stateDb.Close()
 
-	zkProverURI := testutils.GetEnv("ZKPROVER_URI", "localhost")
+	zkProverURI := testutils.GetEnv("ZKPROVER_URI", "54.170.178.97")
```

Comment:
We should keep localhost here
```suggestion
	zkProverURI := testutils.GetEnv("ZKPROVER_URI", "localhost")
```

Reply:
Yes, I should have not pushed that